### PR TITLE
Translation: Update Georgian translation

### PR DESCRIPTION
**Pull request by NorwayFun**

### CODE REVIEW

**Comment by NorwayFun** [comment]
yes

**Review by rousskov — COMMENTED** [review]
Thank you for submitting these translation improvements! I am not qualified to review them, but somebody from the language team will probably do that soon. Please do not be alarmed if it takes them a few days as we are in the middle of a major release cycle.

File: errors/ka.po
Comment (yadij) [inline errors/ka.po:1315]: Please check the concept translated properly. It appears to be replacing `unhandled` with `raw`.

FYI, this message is supposed to be saying that a critical extension of the TLS syntax has not been correctly processed/handled. Either it was missing, or it was invalid, or it contained value(s) that Squid cannot understand.

File: errors/ka.po
Comment (yadij) [inline errors/ka.po:7]: Please also add your name+email as used here to the first list in the file `errors/TRANSLATORS`.

**Review by yadij — CHANGES_REQUESTED** [review]
Thank you for this.

There are a few texts which seemed to be mangled when google translate was applied in reverse. That is usually a sign that the concepts are not matching up correctly between the languages.

**Comment by yadij** [comment]
OK to test

**Comment by NorwayFun** [comment]
Hello,
Thanks for comments. Will close PR for now and reopen it once I'll be sure they're good.

**Comment by yadij** [comment]
@NorwayFun for future; if you use a named branch instead of "master" there should be no need to close. Just append updates to the branch and our merge bot will squash it all down when the process is complete. We use github labels to track what action / by who each PR is waiting.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po]: I've reverted Directory back to Directory from Folder. thanks for suggestion.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:231]: Literal translation is "CRTL is not yet valid". Maybe translator does something incorrect, but transaltion is right

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:239]: Words "signing" (a verb" and "signature" are the same word in Georgian. Just checked with specialists, they said it's good word to use.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po]: Also corrected, thanks for pointing out.

**Comment by NorwayFun** [comment]
> @NorwayFun for future; if you use a named branch instead of "master" there should be no need to close. Just append updates to the branch and our merge bot will squash it all down when the process is complete. We use github labels to track what action / by who each PR is waiting.

I do :) 
Now it has to be corrected. 

Temuri

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po]: also corrected.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:526]: corrected.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:584]: Hello. 
Handshake is "starting of communication" in TCP/IP, which is closest translation I could do.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:1031]: ok, corrected to communication.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:1149]: There is no word "ემიტენტი" in Georgian. Word used here is correct.

File: errors/ka.po
Comment (NorwayFun) [reply errors/ka.po:7]: added.

**Review by yadij — APPROVED** [review]

File: errors/ka.po
Comment (rousskov) [inline errors/ka.po:7]: @NorwayFun, when this PR is merged, the scripts that update Squid `CONTRIBUTORS` file will want to add your GitHub account name and email to that file. If that is _not_ what you want, please let us know what name/email you want to be used for CONTRIBUTORs entry instead, and we will handle it manually. For example, we can add the name/email that are used on this Last-Translator line instead of that GitHub-based default. Or we can add just your name (without email) if that is your preference. Or nothing at all! Your call.

Comment (NorwayFun) [reply errors/ka.po:7]: Just add a name, would be enough for everything, I guess.

Comment (rousskov) [reply errors/ka.po:7]: Will do after this PR merges, thank you!

Comment (rousskov) [reply errors/ka.po:7]: Done in #1289.

**Pull request closed without merging**
